cores/esp8266/time.cpp

```diff
@@ -187,26 +187,44 @@ void configTime(int timezone_sec, int daylightOffset_sec, const char* server1, c
     sntp_init();
 }
 
-void setTZ(const char* tz){
-	
+void setTZ(const char* tz)
+{
     char tzram[strlen_P(tz) + 1];
     memcpy_P(tzram, tz, sizeof(tzram));
     setenv("TZ", tzram, 1/*overwrite*/);
     tzset();
 }
 
-void configTime(const char* tz, const char* server1, const char* server2, const char* server3)
+void configTzTime(const char* tz, const char* server1, const char* server2, const char* server3)
```

Comment:
Don't rename a well-known function.  That breaks backwards compatibility for no reason.

Reply:
FYI, it is already [there](https://github.com/esp8266/Arduino/blob/55ef3e73978ad010d44ad7e4c9bd4b4f32455989/cores/esp8266/Arduino.h#L276-L280)